### ISSUE TITLE
Support for multiple property types

### ISSUE THREAD

**Issue by sebastianroming** [web-padawan/api-viewer-element#8]
As I'm defining my properties with different types, the `api-viewer-element` won't recognize the correct type.

For example, I have the following property:
```
@property({type: Boolean, reflect: true}) invalid: boolean|null|undefined = false;
```
which is originally a `boolean`, but also can be `null` or `undefined`. 
The `api-viewer-element` though ignores this and just displays an input text field, value `false` - no checkbox.

**Comment by web-padawan** [comment]
Thanks, this seems like a bug. Will work on a fix to be included to the next release.

**Comment by web-padawan** [comment]
The fix to handle `null` is now released as `0.2.9` version.
